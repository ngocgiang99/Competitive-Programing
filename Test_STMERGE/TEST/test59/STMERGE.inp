2
4 8
916220592 229101365 671060075 791830070 800518525 997158782 713530212 552754715 
677302371 416375587 5067264 143897135 392633223 985798107 540593798 296961095 
929854288 459338649 348242810 228015737 949204577 381766292 433600417 270517017 
452059039 823371583 489812993 431077053 746461744 124575312 166397917 655603257 
7 10
293107899 31750818 983102239 394346424 534063713 569131342 943202566 254514361 232013523 732120529 
15393107 105703283 133935702 44628480 269928045 263198173 316042607 964701169 823755511 560669738 
195631843 851294201 926903255 72826577 576232376 258380459 974689841 199574149 235347761 877957342 
293570923 896238652 582924501 785994151 612517161 597899051 362314344 148964442 598872402 832511175 
542330862 131991180 916913673 170424517 513669551 216278035 310281651 865197338 115082285 191150851 
505722102 158322682 569292846 397910961 560698551 780687836 579831480 234774195 319278106 281049459 
974571790 304930525 599255173 674549173 698740407 757218117 886947229 751476184 974303607 314224959 
