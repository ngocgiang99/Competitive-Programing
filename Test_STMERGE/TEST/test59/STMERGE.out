452059039
484721774
